File: exercises/0x01/4-04.c
/* Funciones
Calcular cuál es el número menor de dos números enteros. */
#include <stdio.h>
int my_function(int a, int b)
{
	if (a == b)
	{
		printf("Los dos números son iguales\n");
	}
	else
	{
		if (a > b)
		{
			printf("El segundo número es menor que el primer número\n");
		}
		else
		{
			printf("El primer número es menor que el segundo número\n");
		}
	}
	return 0;
}
int main()
{
	int a, b;
	printf("Ingrese primer número: ");
	scanf("%d", &a);
	printf("Ingrese segundo número: ");
	scanf("%d", &b);
	my_function(a, b);
	return 0;
}

File: python/0x01/4-04.py
""" Ejercicios de listas y tuplas
Escribir un programa que pregunte al usuario los números ganadores de la
lotería primitiva, los almacene en una lista y los muestre por pantalla
ordenados de menor a mayor. """
def my_code():
    numbers = []
    for i in range(6):
        print("Ingrese un número ganador ")
        number = int(input())
        numbers.append(number)
    numbers.sort()
    print("Los números ganadores son: " + str(numbers))

my_code()